README.md

```diff
@@ -162,7 +162,7 @@ Will make the demo page available on http://localhost:8081/demo/.
 The `sanitizer` is used in order to protect against malicious certificates that could hold XSS attacks.
 It is an overlay of the [xss](https://www.npmjs.com/package/xss) library, since at times, you might want to be able to configure or adapt the whitelist to your own needs.
 
-To modify it, you should modify the index.js file. Then run:
+To modify it, you should modify the index.js file by adding the CSS properties to whitelist to the object `whiteListedCssProperties`, then run:
```

Comment:
I think the CSS part should be a separate paragraph, you can modify index.js for other purposes too

Reply:
Good point